static/templates/base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %}{% endblock %}</title>
+    {% load static %}
```

Comment:
template tags should be on the first line of the file. Move {% load static %} to the top please

Reply:
Please move {% load %} tag to the top